docs/user/tutorials/01-50-send-requests-using-egress.md

```diff
@@ -0,0 +1,229 @@
+# Send requests using Istio Egress
+
+## Prerequisites
+
+* Istio module with egress enabled.
+* To use CLI instruction, you must install [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+  and [curl](https://curl.se/).
+
+### Configuration
+
+1. Export the following values as environment variables:
+
+    ```bash
+    export NAMESPACE={service-namespace}
+    ```
+
+2. Create a new namespace for the sample application.
+    ```bash
+    kubectl create ns $NAMESPACE
+    kubectl label namespace $NAMESPACE istio-injection=enabled --overwrite
+    ```
+
+3. Make sure there is an Istio CR with egress enabled:
+   ```bash
+   kubectl apply -f - <<EOF
+   apiVersion: operator.kyma-project.io/v1alpha2
+   kind: Istio
+   metadata:
+     name: default
+     namespace: kyma-system
+     labels:
+       app.kubernetes.io/name: default
+   spec:
+     components:
+       egressGateway:
+         enabled: true
+   EOF
+   ```
+
+4. Enable additional sidecar logs to see egressGateway being used in requests:
```

Comment:
```suggestion
4. Enable additional sidecar logs to see the egress Gateway being used in requests:
```
tbh, I don't know if it should be egressGateway / egress gateway / or egress Gateway. I used the form 'egress Gateway' throughout the tutorial so it's written in the same way